File: .history/history/day1/userlist.py
#!/usr/bin/env python
from pprint import pprint
users = []

with open('/etc/passwd') as fp:
    for line in fp:
        users.append( line.split(':')[0].title() )

users.sort()

i = 1
for name in users:
    print "%6s  %s" % (i, name)
    i += 1

File: .history/older-than/older/source-example/day1/userlist.pl
#!/usr/bin/env python
import sys
fp = open(sys.argv[1], 'r')
users = []
for line in fp:
    users.append(line.split(':')[0].title())

users.sort()

for (i, n) in enumerate(users):
    i+=1
    print "%5s %s" % (i, n)   
fp.close()
